code/src/Question2.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+const Question2 = ({section, setSection, livingspace,setLivingspace}) => {
+    
+    return(
+      <>
```

Comment:
No need to wrap everything in React Fragment (`</>`) if you return only one element (in your case `div`)